PURL-TYPES.rst

```diff
@@ -81,15 +81,24 @@ conan
 -----
 ``conan`` for Conan C/C++ packages:
 
-- The default repository is ``https://center.conan.io``
-- The ``namespace`` is the user if present
-- The ``name`` is the package name.
-- The ``version`` is the package version.
-- The qualifier ``channel`` must be not empty if namespace is present
+- The default repository is ``https://center.conan.io``.
```

Comment:
BTW @jsteinhofff, how is a non-default repository declared via PURL?

Reply:
Good point, i dont know, should it? I think PURL is actually rather a misnomer, it should be PURI since the location of the packages are at not in it. At least when i look at the examples in the core specification the repository is nowhere given.

I see no way to indicated the repository in the PURL, so mentioning the default is maybe misleading (but it would be also for other package types here in the document). @sschuberth  what do you think?

Reply:
Well, there *are* [examples](https://github.com/package-url/purl-spec#some-purl-examples) for how to specify a non-default Maven repository by leveraging an optional `repository_url` qualifier. I'd simply propose to use the same approach for Conan and document it here.

Reply:
Yes, I think the fact that these example qualifiers are not URL-encoded is simply an oversight.

Reply:
i now added a reference to the repository_url. Actually i restructured it so the default repository is mentioned for the repository_url qualifier.

Reply:
Just to cross reference relevant discussion: https://github.com/package-url/purl-spec/pull/184

I think rfc2396.txt is not applicable and https://datatracker.ietf.org/doc/html/rfc3986#section-3.4 states something different:
> However, as query components are often used to carry identifying information in the form of "key=value" pairs and one frequently used value is a reference to another URI, it is sometimes better for usability to avoid percent-encoding those characters.